packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -686,6 +686,13 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     );
   }
 
+  // TBD
+  // TODO(Piinks): https://github.com/flutter/flutter/issues/100748
+  @override
+  Simulation? updateBallisticAnimation(double initVelocity, double initPosition) {
```

Comment:
From original PR: Why not implement now?

I think this is ok to return to. The NestedScrollView handles ballistic scroll activity very differently. It is also a temperamental widget, so I feel that I agree - we should evaluate and update it separately.

Source: https://github.com/flutter/flutter/pull/100133/files#r844471868

Reply:
Will follow up in https://github.com/flutter/flutter/issues/100748